vspace/src/main/webapp/WEB-INF/views/deleteModal.jsp

```diff
@@ -0,0 +1,49 @@
+<%@ page pageEncoding="UTF-8"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<script>
+	$(document).ready(function() {
+		$('#confirm-delete').on('click', '.btn-ok', function(e) {
+			var url = $(this).data('url');
+			var urlToLoadOnSuccess = $(this).data('urlToLoadOnSuccess');
+			$.ajax({
+				url : url,
+				type : 'DELETE',
+				success : function(response) {
+					console.log(response);
+					if(response == "Success") {
+						window.location.href = urlToLoadOnSuccess;
+					} else {
+						$('#closeButton').click();
+						$('#errorMessage').find("strong").text(response+", please reload the page and try again");
+						$('#errorMessage').show();							
+					}
+				}
+			});
+		});
+		$('#confirm-delete').on('show.bs.modal', function(e) {
+			var data = $(e.relatedTarget).data();
+			console.log(data);
+			$('.btn-ok', this).data('url', data.url);
+			$('.btn-ok', this).data('urlToLoadOnSuccess', data.callOnSuccess);
+		});
+	});
+</script>
+<div class="modal fade" id="confirm-delete" tabindex="-1" role="dialog"
+	aria-labelledby="myModalLabel" aria-hidden="true">
+	<div class="modal-dialog">
+		<div class="modal-content">
+			<div class="modal-header">
+				<button type="button" class="close" data-dismiss="modal"
+					aria-hidden="true">×</button>
+				<h4 class="modal-title" id="myModalLabel"></h4>
```

Comment:
let's change those default ids. if everybody leaves then when copying the code from bootstrap, we'll end up with duplicate ids.

Reply:
Also, let's add a heading (e.g. "Confirm Space Deletion")